docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-pokeshop.md

```diff
@@ -0,0 +1,327 @@
+        target: /app/provisioning.yaml
+    ports:
+      - 11633:11633
+    command: --provisioning-file /app/provisioning.yaml
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    depends_on:
+      postgres:
+        condition: service_healthy
+      adot-collector:
+        condition: service_started
+    healthcheck:
+      test: ["CMD", "wget", "--spider", "localhost:11633"]
+      interval: 1s
+      timeout: 3s
+      retries: 60
+
+  postgres:
+    image: postgres:14
+    environment:
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+    healthcheck:
+      test: pg_isready -U "$$POSTGRES_USER" -d "$$POSTGRES_DB"
+      interval: 1s
+      timeout: 5s
+      retries: 60
+    ports:
+      - 5432:5432
+
+  adot-collector:
+    image: amazon/aws-otel-collector:latest
+    command:
+      - "--config"
+      - "/otel-local-config.yaml"
+    volumes:
+      - ./collector.config.yaml:/otel-local-config.yaml
+    environment:
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+      AWS_SESSION_TOKEN: ${AWS_SESSION_TOKEN}
+      AWS_REGION: ${AWS_REGION}
+    ports:
+      - 4317:4317
+      - 2000:2000
+```
+
+Tracetest depends on Postgres and the ADOT collector. Tracetest requires config files to be loaded via a volume. The volumes are mapped from the root directory into the `root` directory and the respective config files.
+
+The `tracetest.config.yaml` file contains the basic setup of connecting Tracetest to the Postgres instance.
+
+```yaml
+postgres:
+  host: postgres
+  user: postgres
+  password: postgres
+  port: 5432
+  dbname: postgres
+  params: sslmode=disable
+
+poolingConfig:
+  maxWaitTimeForTrace: 10m
+  retryDelay: 5s
+
+googleAnalytics:
+  enabled: true
+```
+
+The `tracetest.provision.yaml` file definines the trace data store, set to AWS X-Ray, meaning the traces will be stored in X-Ray and Tracetest will fetch them from X-Ray when running tests.
+
+But how does Tracetest fetch traces?
+
+Tracetest uses the golang [AWS-SDK](https://aws.amazon.com/sdk-for-go/) library to pull to fetch trace data.
+
+```yaml
+dataStore:
+  type: awsxray
+  awsxray:
+    accessKeyId: <your-accessKeyId>
+    secretAccessKey: <your-secretAccessKey>
+    sessionToken: <your-session-token>
+    region: "us-west-2"
+```
+
+How do traces reach AWS X-Ray?
+
+The Pokeshop API code uses the native Node.js OpenTelemetry modules which sends information to the ADOT Collector to be processed and then sent to the configured AWS X-Ray SaaS.
+
+## Run Both the Pokeshop API and Tracetest
+
+To start both the Pokeshop API and Tracetest, we will run this command:
+
+```bash
+docker-compose up
+```
+
+This will start your Tracetest instance on `http://localhost:11633/`. Go ahead and open it up.
+Start creating tests! Make sure to use the `http://demo-api:8081/` url in your test creation, because your Pokeshop API and Tracetest are in the same network.
```

Comment:
```suggestion
Make sure to use the `http://demo-api:8081/` url in your test creation, because your Pokeshop API and Tracetest are in the same network.
```